mage_ai/frontend/tests/pages.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { test, expect } from './base';
+
+test('ensure all pages main pages load', async ({ page }) => {
+  async function navigateToAndWaitTilLoaded(name: string) {
+    await page.locator('div[class*="indexstyle__VerticalNavigationStyleComponent"] > div').hover();
+    await page.getByRole('link', { name: name }).click();
+    await page.waitForLoadState('domcontentloaded');
```

Comment:
@edmondwinston Did you try testing out intentionally including a client error (e.g. including `throw new Error('testing page load err');`) on any of the pages and making sure the test failed due to the error?

I tried including that `throw new Error` snippet on the Files page, but the tests were still passing. If I go to the Files page, this error below will pop up, so I would expect the tests to not all pass.
<img width="870" alt="image" src="https://github.com/mage-ai/mage-ai/assets/78053898/aafcbe39-27eb-4d55-8ffd-66d901e54e41">

Reply:
We might need to include an additional `await...toBeVisible` check to make sure the page we navigate to in the test has time to load even further to make the test more consistent. Sometimes when I ran it with the error page, the tests passed and another time, they failed (as intended).

Reply:
![image](https://github.com/mage-ai/mage-ai/assets/132081506/140ea045-f2a0-425e-9a66-4173ad90c8d5)

Tried locally, it seems to work on my side. Maybe it was a caching issue? Also added a `toBeVisible` call for a page's title on the breadcrumb.